sdk/keyvault/azure_security_keyvault_certificates/src/clients.rs

```diff
@@ -92,6 +95,12 @@ impl CertificateClient {
         let certificate_name = certificate_name.to_owned();
         let parameters: Body = parameters.into();
 
+        let mut ctx = options.method_options.context;
+        let span = create_public_api_span(&ctx, None, None);
```

Comment:
Anything authors have to do will be forgotten. The attributes, at least, are one-liners. Why can't this be driven from there?

Reply:
Or in the policy? I don't understand why this is even needed. I thought the whole point was that when any client methods that called other client methods, only the outermost client method telemeters.

Reply:
The policy is a part of the pipeline. If the pipeline is executed multiple times, then the policy creates multiple top level spans.

But the whole point of this is to have *ONE* top level span. You want one `createCertificate` span with multiple HTTP API spans underneath it, since that's the logical structure of the operation being performed - the client only called `createCertificate` once, so we should only have a single span for that operation.